p2p/types/peers_view_test.go

```diff
@@ -1,9 +1,123 @@
 package types
 
 import (
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 )
 
+func TestSortedPeersView_Add_Remove(t *testing.T) {
+	testCases := []struct {
+		name          string
+		selfAddr      string
+		addAddrs      string
+		removeAddrs   string
+		expectedAddrs string
+	}{
+		{
+			"highest self address",
+			"A",
+			"BC",
+			"",
+			"ABC",
+		},
```

Comment:
I understand that cool people don't use VSCode but other editors that show a "hint" about the name of the variables from the context. :)

That said, I don't have a strong opinion, feel free to ignore completely but I would consider making the test cases definitions more verbose, I bothered writing this just because all the arguments are strings. This way I don't have to scroll and read what is argument at index 2 if that makes sense.

Something like this (and below): 

```suggestion
		{
			name:          "highest self address",
			selfAddr:      "A",
			addAddrs:      "BC",
			removeAddrs:   "",
			expectedAddrs: "ABC",
		},
```

If it sounds overkill, I get it. Ignore.

Reply:
Great suggestion :100: , thank you for making me aware of this! :eyes:

Reply:
> I understand that cool people don't use VSCode but other editors that show a "hint" about the name of the variables from the context. :)

On the contrary: cool people DO use VSCode.

But, I do support this change!